route/build/src/var_lookup.f90

```diff
@@ -307,11 +318,14 @@ MODULE var_lookup
  type(iLook_PFAF)     ,public,parameter :: ixPFAF      = iLook_PFAF     ( 1)
  type(iLook_RFLX)     ,public,parameter :: ixRFLX      = iLook_RFLX     ( 1, 2, 3, 4, 5, 6, 7, 8, 9,10, &
                                                                          11,12,13,14,15,16,17,18,19,20, &
-                                                                         21,22,23,24,25,26,27,28)
+                                                                         21,22,23,24,25,26,27,28,29,30, &
```

Comment:
What is the meaning of these numbers?

Reply:
At the top of this F90, there is a comment -` ! defines named variables used to index array elements`, but may not be clear?  this can be revised. it is an index of the named variable.